frontend/src/components/utils/Search.jsx

```diff
@@ -1,141 +1,142 @@
+                  />
+                </InputGroup.Text>
+              </InputGroup.Prepend>
+            )}
+            <FormControl
+              placeholder="Search for Account ID, Txn hash, Block hash, or Block height"
+              aria-label="Search"
+              aria-describedby="search"
+              onChange={this.handleSearchValueChange}
+              className="search-field"
+            />
+            {this.props.dashboard && (
+              <Button type="submit" variant="info" className="button-search">
                 Search
-              </span>
-            ) : (
-              <img
-                src="/static/images/icon-search.svg"
-                className="search-icon"
-              />
+              </Button>
             )}
-          </Col>
+          </InputGroup>
         </Row>
-      </Col>
-      <style jsx global>{`
-        .search {
-          border-radius: 25px;
-          border: solid 2px #e6e6e6;
-          background-color: #f8f8f8;
-          transition: all 0.3s;
-          overflow: hidden;
-        }
-
-        .search-icon-border {
-          border-left: solid 1px #e6e6e6;
-          cursor: pointer;
-          background: rgba(0, 0, 0, 0.1);
-          padding-top: 4px;
-          padding-bottom: 4px;
-        }
-
-        .search-text {
-          font-family: BentonSans;
-          font-size: 14px;
-          color: #999999;
-          outline: none;
-          border: 1px solid #fff;
-          background-color: #f8f8f8;
-          font-weight: 100;
-          padding-top: 5px;
-          padding-bottom: 5px;
-        }
-
-        .search-text::placeholder {
-          color: #999999;
-          opacity: 1; /* Firefox */
-        }
-      `}</style>
-    </Row>
-  );
-};
+        <style jsx global>{`
+          .search-box {
+            background: white;
+            width: ${this.props.dashboard ? "740px" : "520px"} !important;
+            max-width: 100%;
+            height: ${this.props.dashboard ? "49px" : "40px"} !important;
+            margin: auto;
+          }
+
+          @media (max-width: 1000px) {
+            .search-box {
+              width: 100%;
+            }
+          }
+
+          .input-group-text {
+            background: #fafafa;
+            border: 2px solid #eaebeb;
+            border-right: none;
+            border-radius: 8px;
+            height: 100%;
+          }
+
+          .search-field {
+            background: ${this.props.dashboard ? "#FFFFFF" : "#FAFAFA"};
+            border: 2px solid #eaebeb;
+            border-radius: 0.25rem;
+            border-left: ${this.props.dashboard ? "2px solid #eaebeb" : "none"};
+          }
+
+          .search-field:hover {
+            background: #f8f9fb;
+            border: 2px solid #cdcfd1;
+          }
+
+          .search-field:disabled,
+          .search-field[disabled] {
+            background: #eaebeb;
+            border: 1px solid #eaebeb;
+          }
+
+          .button-search {
```

Comment:
Please, fix the `:hover` button background. It is currently somewhat green:
![image](https://user-images.githubusercontent.com/304265/104751480-325fff00-5734-11eb-88c0-6913e00be63e.png)